quiche/src/recovery/mod.rs

```diff
@@ -1525,6 +1537,201 @@ mod tests {
         );
     }
 
+    // TODO: `congestion` and `gcongestion` behave differently. That might be ok
+    // given the different algorithms but it would be ideal to merge and share
+    // the logic.
```

Comment:
Please cleanup this TODO text.  It is hard to follow.

// TODO: Implement `enable_relaxed_loss_threshold` and enable this test for the `congestion` branch.